src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx

```diff
@@ -59,16 +38,6 @@ const MeetingInfo = ({
         return result.data;
     };
```

Comment:
Maybe use the getMeetingByID function from getRequests file or create it there and import it here